Modules/IO/GDCM/test/itkGDCMLoadImageSpacingTest.cxx

```diff
@@ -34,6 +35,11 @@
 // 1,1,1 for the spacing.
 // The patch that this test is part of has a fallback for the case of
 // the MediaStorageTypes for which gdcm has trouble with the spacing tag.
+//
+// expected spacing[0]=0.178038 spacing[1]=0.174924
```

Comment:
Those comments do not make sense to me.

Reply:
@malaterre  test file is still the old one, ultrasound with pixel spacing, it a reminder about spacing, will be updated later, AFAIK